lib/api/channel.js

```diff
@@ -32,15 +34,21 @@ export default class Channel {
     }
     return this._messageHandlers[method].attach(handler)
   }
-
+  /* eslint-disable max-len */
```

Comment:
This and following comments in the file are artefacts of me trying to understand the code. They will be removed before I open a clean PR. They are basically "notes" for my understanding.